spec/dummy/spec/requests/auth_request_spec.rb

```diff
@@ -17,7 +17,7 @@
 
   let(:bypass_auth) { false }
   let(:identity_url) { 'https://my.example.com' }
-  let(:host_url) { 'https://example.com' }
+  let(:host_url) { 'https://www.example.com' }
```

Comment:
Is there any reason for the addition of the subdomain here? (other than perhaps to match the use of `www.example.com` in a later test?)

Reply:
Rails uses `www.example.com` as the request host in tests.  Previously we weren't checking the redirect was working on log in (it would have failed).

The test is to see if `redirect_url.host == request.host`.

It would be better to set host_url so that we don't care what hostname rails uses as its test hostname, but I wasn't sure how to do that.